types/font-carrier/index.d.ts

```diff
@@ -0,0 +1,124 @@
+// Type definitions for font-carrier 0.3
+// Project: https://github.com/purplebamboo/font-carrier#readme
+// Definitions by: zfitness <https://github.com/zFitness>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+// tslint:disable-next-line: no-declare-current-package no-single-declare-module
```

Comment:
You should just be able to delete this `declare module` wrapper and this lint exception.

Reply:
ok

Reply:
Now , i dont want to export Base<T>, but  it reported an error： class Base<T>
Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier

Reply:
Just add the `declare`.

Reply:
It's a small problem， please merge